config/routes.rb

```diff
@@ -0,0 +1,16 @@
+Rails.application.routes.draw do
+  root 'works#main'
+
+  post 'sessions/login', to: 'sessions#login', as: 'login'
+  get 'sessions/login', to: 'sessions#new'
+  delete 'sessions/destroy', to: 'sessions#destroy', as: 'logout'
+
+  get 'works/main', to: 'works#main'
```

Comment:
Since you have `sessions#main` being routed to by the root_path, you don't really need this route.